core-api/src/main/java/com/optimizely/ab/config/Experiment.java

```diff
@@ -56,19 +57,21 @@
     public Experiment(@JsonProperty("id") String id,
                       @JsonProperty("key") String key,
                       @JsonProperty("status") String status,
+                      @JsonProperty("layerId") String layerId,
                       @JsonProperty("audienceIds") List<String> audienceIds,
                       @JsonProperty("variations") List<Variation> variations,
                       @JsonProperty("forcedVariations") Map<String, String> userIdToVariationKeyMap,
                       @JsonProperty("trafficAllocation") List<TrafficAllocation> trafficAllocation) {
-        this(id, key, status, audienceIds, variations, userIdToVariationKeyMap, trafficAllocation, "");
+        this(id, key, status, layerId, audienceIds, variations, userIdToVariationKeyMap, trafficAllocation, "");
```

Comment:
Would you want to add a comment that emphasizes `layerId` is for v2 JSON datafile only?